test/corpus/attributes.txt

```diff
@@ -79,10 +79,9 @@ class Zzz {
       (attribute
         name: (identifier)
         (attribute_argument_list
-          (attribute_argument
-            (member_access_expression
-              expression: (identifier)
-              name: (identifier))))))
+          (member_access_expression
```

Comment:
Maybe put back attribute_argument back between these two for consistency/non-breaking.